test/IntegrationTests/TicketService/test_delete_custom_field.py

```diff
@@ -0,0 +1,219 @@
+import unittest
+from http import HTTPStatus
+import requests
+import logging
+
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(levelname)s - %(message)s'
+)
+logger = logging.getLogger(__name__)
+
+class TestDeleteCustomFieldAPI(unittest.TestCase):
+    BASE_URL = "http://localhost:8000"
```

Comment:
Use Env variable for 8000 

Use CHAT_SERVICE_PORT

Use env variables for all new files